src/index.ts

```diff
@@ -1,4 +1,117 @@
-export default () => {
-  // eslint-disable-next-line no-console
-  console.log('https://www.youtube.com/watch?v=dQw4w9WgXcQ')
-}
+/* eslint-disable @typescript-eslint/naming-convention */
+
+// eslint-disable-next-line import/no-internal-modules
+require('@rushstack/eslint-patch/modern-module-resolution');
```

Comment:
this looks like a workaround for me that I'd like to prevent. can you elaborate on why it's needed?